### PR TITLE
Fix: Fixed Box shadow of the cards.

### PR DESCRIPTION
**Pull request by zoef-yt**
Card changes

Also having badges on the cart and wishlist icon inside the homepage was not making sense as users still have logged it so changed it too.

### CODE REVIEW

**Review by amartya-rs — APPROVED** [review]
Good to merge.👍

**Review by srishtigautam21 — APPROVED** [review]
Great you followed the suggestions of vikram  and as a reviewer now i can better your code....good work

**Review by zoyatanweer — APPROVED** [review]
good to merge.

**Review by varun13169 — APPROVED** [review]

**Review by purohitdheeraj — APPROVED** [review]
Amazing work Zoef! I have stopped playing games after the 12th, but now after watching your e-commerce would definitely try out some games 😃

File: index.html [hunk 1]
```diff
@@ -15,55 +15,49 @@
...
-							fill="currentColor"
-						></path>
-						<path
-							d="M96 216a16 16 0 1 1-16-16a16 16 0 0 1 16 16zm88-16a16 16 0 1 0 16 16a16 16 0 0 0-16-16zm45.4-125.8L203 166.6a24.1 24.1 0 0 1-23.1 17.4H84.1A24.1 24.1 0 0 1 61 166.6L34.6 74.3v-.2L24.8 40H8a8 8 0 0 1 0-16h16.8a16.1 16.1 0 0 1 15.4 11.6L48.3 64h173.4a8 8 0 0 1 7.7 10.2zM211.1 80H52.9l23.5 82.2a8 8 0 0 0 7.7 5.8h95.8a8 8 0 0 0 7.7-5.8z"
-							fill="currentColor"
-						></path>
-					</svg>
-					<div class="badge-icon">1</div>
-				</div>
+				<svg
```
Comment (purohitdheeraj) [inline index.html:18]: just wanted to ask you, why inline SVG for icons?
svgs can be separated into separate files and can be used with img tag
line of codes will be shorter

Comment (zoef-yt) [reply index.html:18]: Adding SVG inside img tag won't change the colour on the hover. That's why.

Comment (bhushannaikaroskar) [reply index.html:18]: yup zoef same question

**Review by bhushannaikaroskar — APPROVED** [review]

**Pull request merged**